routes/conversation.js

```diff
@@ -122,8 +123,32 @@ router.put("/:conversationId", async function (req, res) {
         message: req.body.message
     });
 
+    const uri = process.env.WITAI_URI + encodeURIComponent(req.body.message.message);
+    const auth = 'Bearer ' + process.env.WITAI_KEY;
+
+    let event = {};
+
+    await fetch(uri, {headers: {Authorization: auth}})
+     .then(res => res.json())
+     .then((data) => {
+        if (data.intents.length==1) {
+            if(data.entities.hasOwnProperty('wit$location:location')) 
+              event.location = data.entities['wit$location:location'][0]['value'];
+            if(data.entities.hasOwnProperty('wit$datetime:datetime')) {
+              if(data.entities['wit$datetime:datetime'][0].hasOwnProperty('from')) 
+                event.datetime = new Date(data.entities['wit$datetime:datetime'][0]['from']['value']);
+              else
+                event.datetime = new Date(data.entities['wit$datetime:datetime'][0]['values'][0]['value']);
+            }
+            if(data.entities.hasOwnProperty('purpose:purpose'))
+              event.purpose = (data.entities['purpose:purpose'][0]['value']);
+            event.intent = data.intents[0]['name'];
+          }
+    });
+
     return res.status(200).json({
-        message: "Message Updated",
+        message: "Message updated",
+        event: event
```

Comment:
Small request: Can we check if events has any fields? Say the ML fails it'd be good to not have any event field included when there wasn't an event detected. Lmk if this works already that way